### PR TITLE
OKTA-680750: Fixed intro modal recurrence in home page and changed Auth0 logo in the intro modal

### PR DESCRIPTION
**Pull request by ishan-krishna-okta**
## Description:
- **What's changed?** Fixed intro modal recurrence in home page and changed Auth0 logo in the intro modal
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-680750](https://oktainc.atlassian.net/browse/OKTA-680750)

### Partially resolves:
* [OKTA-681515](https://oktainc.atlassian.net/browse/OKTA-681515)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/.vuepress/public/img/dialog/auth.svg](https://github.com/okta/okta-developer-docs/blob/15070acdbbde17af7bc36d02cf62b95ef25efa18/packages%2F%40okta%2Fvuepress-site%2F.vuepress%2Fpublic%2Fimg%2Fdialog%2Fauth.svg) | 100 | 0 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5d46766c-35f0-4f90-ad91-4398a4cb09df) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.34629228-b166-4801-b573-a8e68c01ba59?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by ishan-krishna-okta** [comment]
Preview link - https://659e4157e0eeffa665a2f5c8--reverent-murdock-829d24.netlify.app/

File: packages/@okta/vuepress-theme-prose/components/Dialog.vue [hunk 1]
```diff
@@ -99,29 +99,54 @@
...
+        <label for="dont-show-again-checkbox">
+          Don't show this again
+        </label>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
+import storage from '../util/localStorage';
```
Comment (emikhasyak-okta) [inline packages/@okta/vuepress-theme-prose/components/Dialog.vue:117]: nit: indentation should be the same as for `export default {` (2 spaces)

**Review by emikhasyak-okta — APPROVED** [review]
Great job! I left a very minor nit, looks good otherwise 🚢

**Pull request merged**